_docs/install-and-upgrade.md

```diff
@@ -11,7 +11,7 @@ order: 2
 
 First, make sure latest [Node.js](https://nodejs.org/) <abbr title="Long Term Support">LTS</abbr> version or latest current release is installed
 with `dpkg -s nodejs | grep '^Version'` and `node --version`. If not, follow the instructions given on the
-[official documentation](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions).
+[official documentation](https://github.com/nodesource/distributions).
```

Comment:
you can link directly to the section of interest: https://github.com/nodesource/distributions?tab=readme-ov-file#debian-and-ubuntu-based-distributions